setup.py

```diff
@@ -722,7 +725,8 @@ def detect_modules(self):
         # pwd(3)
         exts.append( Extension('pwd', ['pwdmodule.c']) )
         # grp(3)
-        exts.append( Extension('grp', ['grpmodule.c']) )
+        if 'vxworks' not in host_platform:
```

Comment:
Can you please add a top level "VXWORKS" constant?

```
VXWORKS = ('vxworks' in host_platform)
```

And then modify "if" to use it, example:

```
if not VXWORKS:
```